sfs/fd/source.py

```diff
@@ -224,6 +224,69 @@ def point_dipole(omega, x0, n0, grid, *, c=None):
     return 1 / (4 * _np.pi) * (1j * k + 1 / r) * _np.inner(offset, n0) / \
         _np.power(r, 2) * _np.exp(-1j * k * r)
 
+def point_circular_piston(omega, x0, n0, grid, R, *, c=None):
+    r"""Point source with circular piston far-field directivity.
+
+    Parameters
+    ----------
+    omega : float
+        Frequency of source.
+    x0 : (3,) array_like
+        Position of source.
+    n0 : (3,) array_like
+        Normal vector (direction) of the circular piston.
+    grid : triple of array_like
+        The grid that is used for the sound field calculations.
+        See `sfs.util.xyz_grid()`.
+    R : float
+        Radius of circular piston.
+    c : float, optional
+        Speed of sound.
+    Returns
+    -------
+    numpy.ndarray
+        Sound pressure at positions given by *grid*.
+
+    Notes
+    -----
+    .. math::
+
+        G(\x-\x_0,\w) = \frac{1}{2\pi} 
+            \frac{J_1(\wc R \sin(\Theta))}{\wc R \sin(\Theta)}
+            \frac{\e{-\i\wc|\x-\x_0|}}{|\x-\x_0|}
+    with 
+
+    .. math::
+
+        \cos(\Theta) =
```

Comment:
Sorry if this is a naive question, but why are you mentioning `cos()` here when `sin()` is used in the equation above?

Reply:
It's because, the cosine can be expressed by the scalar product of the involved vectors, while the sine has to be expressed ``sqrt(1 - cos^2())``, which is also not fully correct because it could also be ``-sqrt(1 - cos^2())``.

Reply:
I see. Probably you could add those explanations to the docs then, to avoid people thinking this is a mistake?